lib/generators/shopify_app/shopify_app_generator.rb

```diff
@@ -2,8 +2,13 @@ module ShopifyApp
   module Generators
     class ShopifyAppGenerator < Rails::Generators::Base
 
+      def initialize(args, *options)
+        @opts = options.first
+        super(args, *options)
```

Comment:
yeah - the code is kind of weird looking though so I want to leave this anyways.